views/index.html

```diff
@@ -95,31 +95,29 @@ <h1 class="animated fadeInUp">CVR</h1>
 <code>
 #!/bin/bash
 
-# Required Travis config:
-# - Add the repo token in the variable CVR_TOKEN as a secure property in the Travis configuration
-# - LCOV_OUTPUT below must be the file path to the coverage output
-
+# Get the output folder for where the output is going to live
+COBERTURA_FILE="${TRAVIS_BUILD_DIR}/${FASTLANE_FOLDER}/${TEST_OUTPUT_FOLDER}/${TEST_COVERAGE_FILE}"
```

Comment:
This bit right here is a bit specific to our Fastlane setup rather than Travis in general, so I'm not sure if it belongs in this example.